dcmgr/lib/dcmgr/node_modules/vnet_collector.rb

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+require 'isono'
+require 'ipaddress'
+
+require 'vnet_api_client'
+
+module Dcmgr
+  module NodeModules
+    class VnetCollector < Isono::NodeModules::Base
+      include Isono::NodeModules
+      include Dcmgr::Logger
+
+      initialize_hook do
+        endpoint = Dcmgr::Configurations.dcmgr.features.vnet_endpoint
+        port = Dcmgr::Configurations.dcmgr.features.vnet_endpoint_port
```

Comment:
What happens when there's no `features` in `dcmgr.conf`?

Perhaps it's a good idea to only load this module if the openvnet features are enabled? For example with a check in `bin/collector`.